src/Hedgehog/Gen.fs

```diff
@@ -3,12 +3,17 @@
 open System
 open Hedgehog.Numeric
 
+
 /// A generator for values and shrink trees of type 'a.
 [<Struct>]
 type Gen<'a> =
     | Gen of Random<Tree<'a>>
 
 module Gen =
+
+    // Needed because 'double' is shadowed later
+    let inline private fsharpDouble x = double x
```

Comment:
Can't we just use the full qualified name instead? If we have to do that in more than three places, perhaps we can declare an inline function as the one above. Right now it's been used only in two places.

Reply:
Sure. I didn't find where it was (I looked in `FSharp.Core.Operators`, but it wasn't there), and decided that it wasn't worth the trouble. Do you know the fully qualified name?

Reply:
Nevermind, I found it.